utils/scripts/install-dev.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script sets up a new development node with all dependencies needed to
+# build DAOS from source.
+
+# Requirements:
+# - Run as a non-root user with sudo access.
+
+if [ `whoami` == "root" ]; then
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use $(...) notation instead of legacy backticked `...`. [SC2006]